sagemaker-python-sdk/tensorboard_keras/tensorboard_keras.ipynb

```diff
@@ -48,7 +48,7 @@
    "source": [
     "import sys\n",
     "!{sys.executable} -m pip install --upgrade pip\n",
-    "!{sys.executable} -m pip install --upgrade grpcio sagemaker==1.60.* awscli==1.* tensorflow==2.2.* tensorboard_plugin_profile==2.2.*"
+    "!{sys.executable} -m pip install --upgrade grpcio sagemaker==2.23.* awscli==1.* tensorflow==2.2.* tensorboard_plugin_profile==2.2.*"
```

Comment:
Please remove the sagemaker install.

Reply:
TensorBoard works out of the box, as it comes preinstalled in DLC. No need to install anything.
Pushed the code without the sagemaker/tensorflow/tensorboard install. Used conda_tensorflow2_p36 instead.